skins/oasis/modules/templates/PageHeader_Index.php

```diff
@@ -7,12 +7,20 @@
 	}
 }
 
+if ( $showCuratedContentToolBtn ) {
+	$curatedContentToolButton = '<a id="CuratedContentTool" class="wikia-button">' . wfMsg( 'wikiacuratedcontent-edit-mobile-main-page' ) . '</a>';
```

Comment:
I used it because it's already being used in this file. Do you think I should change all occurences of it? That would probably mean regression testing.